package.json

```diff
@@ -13,12 +13,14 @@
     "clean": "touch index.ts && npm run build | grep TSFILE | cut -d' ' -f 2 | xargs rm",
     "build-ts": "tsc",
     "services": "npx kourou app:start-services",
+    "jest": "jest",
```

Comment:
You should take a look to the jest.config to only test `function-tests` folder for now

Reply:
For now all the unit tests are launched with this command

Reply:
The command `jest:test` only launches tests within the `test-functional` folder
The command `jest` is here so you can launch test individualy using `npm run jest <file>` if you only want to execute a specific test